runtime/dotnet/azurewebapp/Scripts/deploy.ps1

```diff
@@ -63,37 +63,46 @@ dotnet user-secrets init
 $publishFolder = $(Join-Path $projFolder 'bin\Release\netcoreapp3.1')
 dotnet publish -c release -o $publishFolder -v q > $logFile
 
-
 # Copy bot files to running folder
 $remoteBotPath = $(Join-Path $publishFolder "ComposerDialogs")
 Remove-Item $remoteBotPath -Recurse -ErrorAction Ignore
 
-
 if (-not $botPath) {
 	# If don't provide bot path, then try to copy all dialogs except the runtime folder in parent folder to the publishing folder (bin\Realse\ Folder)
-	$botPath = '..'
+	$botPath = '../..'
 }
 
 $botPath = $(Join-Path $botPath '*')
 Write-Host "Publishing dialogs from external bot project: $($botPath)"
 Copy-Item -Path (Get-Item -Path $botPath -Exclude ('runtime', 'generated')).FullName -Destination $remoteBotPath -Recurse -Force -Container
 
 # Try to get luis config from appsettings
-$settings = Get-Content $(Join-Path $projFolder appsettings.deployment.json) | ConvertFrom-Json
```

Comment:
appsettings.deployment.json has no effect to the published bot, should be appsettings.json